command/operator_raft_state.go

```diff
@@ -0,0 +1,94 @@
+package command
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/hashicorp/nomad/helper/raftutil"
+	"github.com/posener/complete"
+)
+
+type OperatorRaftStateCommand struct {
+	Meta
+}
+
+func (c *OperatorRaftStateCommand) Help() string {
+	helpText := `
+Usage: nomad operator raft _state <path to nomad data dir>
+
+  Display the log entries persisted in data dir in json form
+
+Options:
+
+  --last-index=<last_index>
```

Comment:
This may be my silliest edit to date :sweat_smile: 

```suggestion
  -last-index=<last_index>
```